### PR TITLE
docs: Add docs for running the starship website locally

### LINKED ISSUE #543: Write Documentation on Running Vuepress Web Page Locally
#### Is your feature request related to a problem? Please describe.
Running Vuepress locally can be made clearer to newcomers of the framework who would like to contribute to designing the website.

#### Describe the solution you'd like
Create documentation for web designers to run VuePress locally, which can include:
- vuepress needs to be installed globally
- npx keyword must come before vuepress dev, and command works only in docs folder
- npx uses the installed library in the project if its available/temporary installs it

#### Describe alternatives you've considered
Since VuePress is an important part of the project, I could not think of any alternatives.

### PR DESCRIPTION
**Pull request by CSumm**
#### Description
Implement setup documentation for running VuePress site locally inside of Contributing.md.

#### Motivation and Context
It clearly demonstrates how to run the VuePress website for those contributing to the visual web design.
Closes #543 in particular

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### How Has This Been Tested?

Opened and tested changes on Windows 10 in Visual Studio Code.

- [ ] I have tested using **MacOS**
- [ ] I have tested using **Linux**
- [x] I have tested using **Windows**

#### Checklist:

- [x] I have updated the documentation accordingly.
- [ ] I have updated the tests accordingly.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -90,3 +90,19 @@ The reason for having _both_ the Dockerfile as well as the Azure Pipelines confi
...
+
+## Locally Running The VuePress Website
+
+If you are contributing to the design of Starship's website, the following section will help you get started.
+
+### Setup
+
+After cloning the project, you can do the following to run the VuePress website on your local machine
+
+1. cd into the /docs directory.
```
Comment (matchai) [inline CONTRIBUTING.md]: Usually, commands and directories are code blocked. 🙂 

```suggestion
1. `cd` into the `/docs` directory.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -90,3 +90,19 @@ The reason for having _both_ the Dockerfile as well as the Azure Pipelines confi
...
+2. Install the project dependencies:
+`npm install`
```
Comment (matchai) [inline CONTRIBUTING.md]: Let's make these lines full code-fences.

````suggestion
```
$ npm install
```
````

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -90,3 +90,19 @@ The reason for having _both_ the Dockerfile as well as the Azure Pipelines confi
...
+3. Start the project in development mode:
+`npm run dev`
```
Comment (matchai) [inline CONTRIBUTING.md]:
````suggestion
```
$ npm run dev
```
````

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -90,3 +90,19 @@ The reason for having _both_ the Dockerfile as well as the Azure Pipelines confi
...
+
+Once setup is complete, you can refer to VuePress documentation on the actual design implementation [here](https://v1.vuepress.vuejs.org/guide/).
```
Comment (matchai) [inline CONTRIBUTING.md]: "here" makes for a small click target. No harm in providing the full URL, in my opinion. 🙂 

```suggestion
Once setup is complete, you can refer to VuePress documentation on the actual design implementation here: https://v1.vuepress.vuejs.org/guide/
```

File: CONTRIBUTING.md (hunk 1 above)
Comment (CSumm) [reply CONTRIBUTING.md]: Perfect. Now I know!

**Comment by matchai** [comment]
@all-contributors Please add @CSumm for docs.

**Pull request merged**